HTMLToQPDF/Components/ParagraphComponent.cs

```diff
@@ -171,5 +172,42 @@ private TextStyle ParseStyleString(string styleString, TextStyle textStyle)
             }
             return textStyle;
         }
+
+        private string GetAlignmentAttribute(HtmlNode element)
+        {
+            foreach (var attr in element.Attributes)
+            {
+                var value = attr.Value;
+                if (attr.Name == "class")
+                {
+                    List<string> classes = new List<string>(value.Split(' '));
+                    foreach (string className in classes)
+                    {
+                        if (className == "ql-align-right" || className == "ql-align-center" || className == "ql-align-justify")
+                        {
+                            return className;
+                        }
+                    }
+                }
+                else if (attr.Name == "style")
+                {
+                    List<string> styles = new List<string>(value.Split(';'));
+                    foreach (var s in styles)
+                    {
+                        var split = s.Split(':');
+                        if (split.Length == 2)
+                        {
+                            var styleType = split[0].Trim();
+                            var styleValue = split[1].Trim();
```

Comment:
This would be faster if the assignment was inside the if statement.

Reply:
updated in latest commit

Reply:
no i'm referring to deleting line 200 and moving that assignment to line 203. that case the split access and the trim only occur in the if statement on line 201

Reply:
I see what you mean. I will update this in my next PR